.ci.yaml

```diff
@@ -1206,234 +1206,234 @@ targets:
         ["devicelab"]
     scheduler: luci
 
-  - name: Linux flutter_gallery__image_cache_memory
-    builder: Linux flutter_gallery__image_cache_memory
+  - name: Linux_android flutter_gallery__image_cache_memory
+    builder: Linux_android flutter_gallery__image_cache_memory
     presubmit: false
     properties:
       tags: >
         ["devicelab"]
     scheduler: luci
 
-  - name: Linux flutter_gallery__memory_nav
-    builder: Linux flutter_gallery__memory_nav
+  - name: Linux_android flutter_gallery__memory_nav
+    builder: Linux_android flutter_gallery__memory_nav
     presubmit: false
     properties:
       tags: >
         ["devicelab"]
     scheduler: luci
 
-  - name: Linux flutter_gallery__start_up
-    builder: Linux flutter_gallery__start_up
+  - name: Linux_android flutter_gallery__start_up
+    builder: Linux_android flutter_gallery__start_up
     presubmit: false
     properties:
       tags: >
         ["devicelab"]
     scheduler: luci
 
-  - name: Linux flutter_gallery__transition_perf_e2e
-    builder: Linux flutter_gallery__transition_perf_e2e
-    bringup: true # Flaky https://github.com/flutter/flutter/issues/83298
+  - name: Linux_android flutter_gallery__transition_perf_e2e
+    builder: Linux_android flutter_gallery__transition_perf_e2e
+    bringup: true
     presubmit: false
     properties:
       tags: >
         ["devicelab"]
     scheduler: luci
 
-  - name: Linux flutter_gallery__transition_perf_hybrid
-    builder: Linux flutter_gallery__transition_perf_hybrid
+  - name: Linux_android flutter_gallery__transition_perf_hybrid
+    builder: Linux_android flutter_gallery__transition_perf_hybrid
     presubmit: false
     properties:
       tags: >
         ["devicelab"]
     scheduler: luci
 
-  - name: Linux flutter_gallery__transition_perf_with_semantics
-    builder: Linux flutter_gallery__transition_perf_with_semantics
+  - name: Linux_android flutter_gallery__transition_perf_with_semantics
+    builder: Linux_android flutter_gallery__transition_perf_with_semantics
     presubmit: false
     properties:
       tags: >
         ["devicelab"]
     scheduler: luci
 
-  - name: Linux flutter_gallery__transition_perf
-    builder: Linux flutter_gallery__transition_perf
+  - name: Linux_android flutter_gallery__transition_perf
+    builder: Linux_android flutter_gallery__transition_perf
     presubmit: false
     properties:
       tags: >
         ["devicelab"]
     scheduler: luci
 
-  - name: Linux flutter_gallery_android__compile
-    builder: Linux flutter_gallery_android__compile
+  - name: Linux_android flutter_gallery_android__compile
+    builder: Linux_android flutter_gallery_android__compile
     presubmit: false
     properties:
       tags: >
         ["devicelab"]
     scheduler: luci
 
-  - name: Linux flutter_gallery_sksl_warmup__transition_perf_e2e
-    builder: Linux flutter_gallery_sksl_warmup__transition_perf_e2e
-    bringup: true # Flaky https://github.com/flutter/flutter/issues/83298
```

Comment:
same here

Reply:
Thanks, re-added! One day my LSC script will support not overriding these :)